### PR TITLE
Fixed lintian warnings

### PR DESCRIPTION
**Pull request by szepeviktor**
Please help me with these C-related ones:

**E:** php5-suhosin-extension: unstripped-binary-or-object usr/lib/php5/20131226/suhosin.so
W: php5-suhosin-extension: [hardening-no-relro](https://wiki.debian.org/Hardening) usr/lib/php5/20131226/suhosin.so

I can't write a [Hello world!](http://rosettacode.org/wiki/Hello_world/Text#C) in C.

### CODE REVIEW

File: pkg/build_deb.sh [hunk 1]
```diff
@@ -81,13 +81,14 @@ fi
...
 mkdir -p $ROOT/DEBIAN
-echo "9" >$ROOT/DEBIAN/compat
+#echo "9" >$ROOT/DEBIAN/compat
 cat >$ROOT/DEBIAN/control <<EOF
 Package: php5-suhosin-extension
 Section: php
 Priority: extra
 Maintainer: Ben Fuhrmannek <ben@sektioneins.de>
 Homepage: http://www.suhosin.org/
+Depends: libc6 (>= 2.4), phpapi-20131226, php5-common, ucf
```
Comment (szepeviktor) [inline pkg/build_deb.sh]: `phpapi-20131226` is jessie specific.

File: pkg/build_deb.sh [hunk 2]
```diff
@@ -81,13 +81,14 @@ fi
 ##
 
 mkdir -p $ROOT/DEBIAN
-echo "9" >$ROOT/DEBIAN/compat
+#echo "9" >$ROOT/DEBIAN/compat
```
Comment (szepeviktor) [inline pkg/build_deb.sh]: `debhelper` is not used, see [COMPATIBILITY LEVELS](http://manpages.debian.org/cgi-bin/man.cgi?query=debhelper&apropos=0&sektion=0&manpath=Debian+8+jessie&format=html&locale=en).

File: pkg/build_deb.sh (hunk 1 above)
Comment (bef) [inline pkg/build_deb.sh]: Added php5-common, which provides phpapi-... and already depends on ucf and libc. This should work for all debian/ubuntu versions, even wheezy.

Comment (szepeviktor) [inline pkg/build_deb.sh]: Thank you.

**Comment by szepeviktor** [comment]
@bef I think this one is useless.
OK to close?

**Comment by bef** [comment]
Sure. I think it is fixed anyway.

**Pull request closed without merging**